overlays/migrate-to-nonroot/redis/redis-store.Deployment.yaml

```diff
@@ -8,7 +8,7 @@ spec:
       initContainers:
         - name: transfer-file-ownership
           image: sourcegraph/alpine:3.10@sha256:4d05cd5669726fc38823e92320659a6d1ef7879e62268adec5df658a0bacf65c
-          command: ["sh", "-c", "chown -R 999:999 /redis-data"]
+          command: ["sh", "-c", "chown -R 999:1000 /redis-data"]
```

Comment:
Oops @uwedeportivo we fixed this here but didn't update base/redis/redis-store.Deployment.yaml